kgforge/specializations/stores/demo_store.py

```diff
@@ -131,6 +132,12 @@ def search(self, resolvers: Optional[List[Resolver]], *filters, **params) -> Lis
         records = self.service.find(conditions)
         return [_to_resource(x) for x in records]
 
+    def _sparql():
```

Comment:
should have the self parameter, same for elastic

Reply:
oh right, keep missing those